pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -277,15 +278,21 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 		if err != nil {
 			return perrors.Wrapf(err, "error checking endpoints %s", sks.Status.PrivateServiceName)
 		}
-		logger.Infof("PA got=%v, want=%v", got, want)
-
+		logger.Infof("PA scale got=%v, want=%v", got, want)
 	}
+
 	err = reportMetrics(pa, want, got)
 	if err != nil {
 		return perrors.Wrap(err, "error reporting metrics")
 	}
 
-	updatePAStatus(pa, want, got)
+	// computeActiveCondition decides if we need to change the SKS mode.
+	if computeActiveCondition(pa, want, got) {
```

Comment:
So part of my problem here is that reading the name of this, I can't tell what the returned bool is supposed to mean.

Reply:
TBH, I think it would help if you named the returned value:

```suggestion
	if changed := computeActiveCondition(pa, want, got); changed {
```

Reply:
Okay, did that. updated the commnet.